src/Backend/Modules/MediaLibrary/Component/StorageProvider/YoutubeStorageProvider.php

```diff
@@ -8,7 +8,7 @@ class YoutubeStorageProvider extends MovieStorageProvider
 {
     public function getIncludeHTML(MediaItem $mediaItem): string
     {
-        return '<iframe width="560" height="315" src="' . $this->includeUrl . $mediaItem->getUrl() . '" frameborder="0" allowfullscreen></iframe>';
+        return '<iframe width="100%" height="100%" src="' . $this->includeUrl . $mediaItem->getUrl() . '" frameborder="0" allowfullscreen></iframe>';
```

Comment:
Does `height="100%"` even work? I'd run this by a frontender

Reply:
it worked when I tried it here. It was full in the gallery and a small player when displayed as a single item.
If there is a better way a frontender may always suggest it
@forkcms/frontenders